beetmoverscript/test/test_work_dir/task_artifact_map.json

```diff
@@ -0,0 +1,137 @@
+{
```

Comment:
This is great, thanks for adding this and the follow-up tests! 🥇  Once declarative artifacts is wrapped up, I think the glorious future should hold integration tests. As in, for each type of beetmover tasks, we should have an integration test for (that includes per product too). Unfortunately, this is something skipped from the very beginning of beetmoverscript implementation so the current logic is great and thanks again for adding these!

Reply:
+1